File: ezSCORES/ezSCORES.Model/CompetitionsTeamsPlayers.cs
using System;
using System.Collections.Generic;
using System.Numerics;
using System.Text;

namespace ezSCORES.Model
{
	public class CompetitionsTeamsPlayers
	{
		public int Id { get; set; }

		public int CompetitionsTeamsId { get; set; }

		public int PlayerId { get; set; }

		public int GoalsTotal { get; set; }

		public bool IsVerified { get; set; }
		public virtual Players Player { get; set; } = null!;
	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/competitionsTeamsPlayers.dart
import 'package:ezscores_mobile/models/players.dart';
import 'package:json_annotation/json_annotation.dart';

part 'competitionsTeamsPlayers.g.dart';

@JsonSerializable()
class CompetitionsTeamsPlayers {
	int? id;
	int? competitionsTeamsId;
	int? playerId;
	int? goalsTotal;
	bool? isVerified;
  Players? player;

	CompetitionsTeamsPlayers();

	factory CompetitionsTeamsPlayers.fromJson(Map<String, dynamic> json) => _$CompetitionsTeamsPlayersFromJson(json);

	Map<String, dynamic> toJson() => _$CompetitionsTeamsPlayersToJson(this);
}
